### DISCUSSION TITLE
Autocomplete symbols in scope on <tab>

### DISCUSSION THREAD

**Discussion by nilsmartel** [babashka/babashka#1256] [Ideas]
when the user presses tab, it would be nice if a completion would be made or selection of candidates would be shown, from the set of symbols in scope.

**Comment by borkdude** [comment]
@nilsmartel Please specify the context? In an editor? In the console? If you are referring to the console: the console behavior mimics what the official Clojure does, which is not too much. If you want to have a better experience I recommend using an nREPL client with the `bb nrepl-server` command.

**Reply by nilsmartel** [reply]
Yeah, this is about the terminal-repl use case.

I will try out the nrepl trick you mentioned, thank you for pointing me in that direction
